pages/about.html

```diff
@@ -0,0 +1,201 @@
+                        organization that was founded in 2015 (Reg. No. 2094 -
+                        PIN. P051557264R) on the premise that every child
+                        deserves access to post-primary education. Since the
+                        inception of Free Primary Education in 2003, the dreams
+                        of many children have been snuffed out because of their
+                        parents/guardians’ inability to afford secondary
+                        education. </p>
+                </div>
+
+                <p>The greatness of humanity is not in being human, but in being
+                    humane. – Mahatma Gandhi
+                </p>
+
+            </section>
+
+        </div>
+        <section class="objectives">
+            <h3>Our Objectives</h3>
+            <hr>
+            <div class="objectives-container">
+                <div class="bg">
+                    <ul class="objective">
+                        <li>
+                            <p>To promote the best interest of children
+                                including health, education and general well
+                                being.</p>
+                        </li>
+                    </ul>
+                </div>
+                <div class="bg">
+                    <ul class="objective" id="values">
+                        <li>
+                            <p>To support children to adulthood by facilitating
+                                their education and instilling in them life
+                                skills.
+                            </p>
+                        </li>
+
+                    </ul>
+
+                </div>
+
+                <div class="bg">
+                    <ul class="objective">
+                        <li>
+                            <p>
+                                To influence children positively and
+                                facilitate them to nature values that will
+                                influence their communities and society
+                                positively.
+                            </p>
+                        </li>
+                    </ul>
+
+                </div>
+
+                <div class="bg">
+                    <ul class="objective">
+                        <li>
+                            <p>
+                                To aid the children in harnessing their talents
+                                and formal skills through sensitisation and
+                                training on the available opportunities for
+                                economic empowerment and development.
+                            </p>
+                        </li>
+                    </ul>
+                </div>
+
+            </div>
+            <button type="button" class="donate-btn">Donate</button>
+
+        </section>
+        <section class="past-events">
+
+            <h3>See past events</h3>
+            <hr>
```

Comment:
this title doesn't appear both on mobile and desktop kindly fixe this problem 

![Capture d’écran du 2023-08-02 00-47-36](https://github.com/NoelLincoln/first-capstone-project/assets/59707859/a1ce3d6b-4720-4d2b-a4cd-cb05acd23d8a)
![Capture d’écran du 2023-08-02 00-47-20](https://github.com/NoelLincoln/first-capstone-project/assets/59707859/23fd00bb-3460-47f8-80b2-f5465e9dd199)